Makefile

```diff
@@ -79,10 +79,17 @@ endif
 
 .PHONY: __bake
 __bake: upload-code upload-data upload-notebooks
-	echo "#!/usr/bin/env bash" > jupyter.sh
-	echo "jupyter notebook --no-browser --ip=0.0.0.0 --allow-root \
-	    --NotebookApp.token= --notebook-dir=/project/notebooks" >> jupyter.sh
-	$(NEURO) cp jupyter.sh $(PROJECT_PATH_STORAGE)/jupyter.sh
+	echo "#!/usr/bin/env bash" > /tmp/jupyter.sh
+	echo "jupyter notebook \
+            --no-browser \
+            --ip=0.0.0.0 \
+            --allow-root \
+            --NotebookApp.token= \
+            --NotebookApp.default_url=/notebooks/project/notebooks/demo.ipynb\
```

Comment:
Shouldn't this URL be `/project/notebooks/demo.ipynb`?

Reply:
not, it will work as it is. Check out the command 
```
neuro run -s cpu-small --pass-config --browse --no-http-auth --http 8888 -n test image://artemyushkovskiy/neuromation-ml-recipe-object-detection:latest bash /project/jupyter.sh
```
I built image `image://artemyushkovskiy/neuromation-ml-recipe-object-detection:latest` manually using `make setup __BAKE_SETUP=yes`